### ISSUE TITLE
request_q4osteam - logout dialog

### ISSUE THREAD

**Issue by seb3773** [seb3773/Q4OSseb#20]
Is it possible to have the logout dialog at this place:
![logoutdlg3](https://github.com/seb3773/Q4OSseb/assets/99963207/fb098b99-c625-4c0e-b44d-6aa28dede4d9)
instead of the center of the screen ? (maybe with an option in config file ?)

**Comment by q4osteam** [comment]
Yes, it should be possible with some code updates, however we will check the code and post back with more info.

**Comment by seb3773** [comment]
Thianks :) This will be very good, it's one of the last details who can make the theme more "credible". 

By the way, I renamed the project, as "Q4osSeb" was a bit egocentric, it was ok when these scripts were only made for my usage, but as I'm trying to improving them for the most, I thought "q4osXpack" is a better name.

As you may have already seen, the scripts are now "Kdialog driven" , I'm trying to make nice menus to make them easier to use. Some parts are not yet "converted" (theme tools for example), but I'm working on it :)
